consensus/cometbft/service/blobreactor/config.go

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: BUSL-1.1
+//
+// Copyright (C) 2025, Berachain Foundation. All rights reserved.
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file of this repository and at www.mariadb.com/bsl11.
+//
+// ANY USE OF THE LICENSED WORK IN VIOLATION OF THIS LICENSE WILL AUTOMATICALLY
+// TERMINATE YOUR RIGHTS UNDER THIS LICENSE FOR THE CURRENT AND ALL OTHER
+// VERSIONS OF THE LICENSED WORK.
+//
+// THIS LICENSE DOES NOT GRANT YOU ANY RIGHT IN ANY TRADEMARK OR LOGO OF
+// LICENSOR OR ITS AFFILIATES (PROVIDED THAT YOU MAY USE A TRADEMARK OR LOGO OF
+// LICENSOR AS EXPRESSLY REQUIRED BY THIS LICENSE).
+//
+// TO THE EXTENT PERMITTED BY APPLICABLE LAW, THE LICENSED WORK IS PROVIDED ON
+// AN "AS IS" BASIS. LICENSOR HEREBY DISCLAIMS ALL WARRANTIES AND CONDITIONS,
+// EXPRESS OR IMPLIED, INCLUDING (WITHOUT LIMITATION) WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, NON-INFRINGEMENT, AND
+// TITLE.
+
+package blobreactor
+
+import "math"
+
+const (
+	// Height at which consensus params are upgraded to use blobreactor
+	blobConsensusUpdateHeight int64 = 0
+
+	// Height to enable blobreactor
+	blobConsensusEnableHeight int64 = math.MaxInt64
+
+	// maxBytes is the maximum size of blob data in bytes for BlobReactor consensus params (e.g., 800KB).
+	blobMaxBytes int64 = 800 * 1024 // 800KB
```

Comment:
note for self: surprised we had to define this quantity. Don't we already enforce explicitly a max size for blobs? If not shouldn't we express it in terms of other quantities?